packages/api-plugin-fulfillment-method-shipping-ups/src/util/validateOrderMethodsups.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @summary Sample dummy function validating the data requirements for this metod
+ * @param {Object} context - Context object
+ * @param {Object} commonOrder - Current order which provide available details to perform validation
+ * @param {Object[]} validationResults - Validation results collected till now
+ * @returns {Object[]} validationResults - with the validation details populated
+ */
+export default function validateOrderMethodsups(context, commonOrder, validationResults = []) {
+  // const { items, shippingAddress } = commonOrder;
```

Comment:
No commented code

Reply:
I had left the line there to show the possibility of getting the item and address details to collect realtime rates from ups
Removed it.